includes/functions/plugins/article-export.php

```diff
@@ -459,30 +545,165 @@ function build_xml() {
 			)
 		);
 
-		//perform dom manipulations
+		// perform dom manipulations
 		$dom = $this->manipulate_dom( $dom );
 
-		//do html_to_xml before adding elements so that the html wrap stuff is removed first
+		// do html_to_xml before adding elements so that the html wrap stuff is removed first
 		$xml_elements = $this->html_to_xml( $dom );
 
+		$elements = apply_filters(
+			__NAMESPACE__ . '\xml_outer_elements',
+			$this->get_outer_elements( $this->article ),
+			$this->article
+		);
+
 		if ( $elements ) {
 			$this->add_outer_elements( $xml_elements, $elements );
 		}
 
 		$this->add_article_attributes( $xml_elements->root_element, $elements );
 
-		//reset global post
+		// reset global post
 		$post = $old;
 
+		$GLOBALS['recentComment'] = $elements['comment'] ? $elements['comment'] : false;
+
 		return $xml_elements;
+	}
+
+	/**
+	 * Get prepared article content
+	 *
+	 * @param object $article WP_Post article
+	 *
+	 * @return string article content
+	 */
+	function get_article_content( $article ) {
+
+		$content = $this->article->post_content;
+
+		if ( $post = get_post( get_post_thumbnail_id( $this->id ) ) ) {
```

Comment:
We should move the assignment of a variable outside of the `if` statement:

```suggestion
		$thumbnail = get_post( get_post_thumbnail_id( $this->id ) );
		if ( $thumbnail ) {
```

Reply:
@dkotter Same as above, code was refactor and no longer exists as it wasn't being used.

Reply:
Updated